src-tools/netmodel/tuntap_test.py

```diff
@@ -59,23 +59,42 @@ def single_test(output_file, packet_size, number_of_welds, number_of_threads):
             if iter > 5 and float(line) != 0:
                 results.append(float(line))
         print(results)
-        output_file.write(str(size))
-        output_file.write('\t')
-        output_file.write(str(sum(results[:-2]) / (len(results[:-2]))))
+        # output_file.write(str(size))
+        # output_file.write('\t')
+        # output_file.write(str(sum(results[:-2]) / (len(results[:-2]))))
+        output_file.write(str(size) + '\t' + str(sum(results[:-2]) / (len(results[:-2]))))
         output_file.write('\t')
         output_file.write(str(number_of_welds))
         output_file.write('\t')
         output_file.write(str(number_of_threads))
         output_file.write('\n')
 
 
+def myrange(low,high,mod):
+  low2 = (low//mod)*mod
+  high2 = (high//mod)*mod
+  return range(low2, high2, mod)
+
+
+def _fail(output):
+    return '\033[91m{}\033[0m'.format(output)
```

Comment:
Little confusing, but once it is not a problem.
If you intend to use colored output more often in the future, consider something similar to `BColors` from `contrib/devtools/git-hist-check.py`